index.html

```diff
@@ -95,16 +95,16 @@ <h2 class="about">About myself</h2>
       <div class="formsection">
         <h3 class="styleh3">CONTACT US</h3>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        <form action="" class="formstyle">
-          <input type="text" placeholder="Text input">
-          <input type="email" placeholder="email">
-          <textarea name="" id="" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/moqrojze" method="POST" class="formstyle">
+          <input type="text" placeholder="Text input" maxlength="30" required>
+          <input type="email" placeholder="email" required>
+          <textarea name="text" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
```

Comment:
Hi team ✋

- [ ] [Optional] You may please make the textArea more identical to the Figma design
- Your design
![your textarea](https://user-images.githubusercontent.com/63917725/177392927-148971bf-33b8-475a-b92e-4b77ff8289d7.PNG)

- Figma design
![figma textarea](https://user-images.githubusercontent.com/63917725/177392969-3a3cd5b7-9542-4f86-a0f6-52f1a65ffd2e.PNG)